.gitignore

```diff
@@ -3,3 +3,4 @@
 build
 out
 .env
+.tmp
```

Comment:
Why do we need to ignore this folder?
Lets not use a common static tmp folder, but rather create per-test temp folders

Reply:
Removed